news/2018-06-23-osu-mapping-olympiad-1-results.md

```diff
@@ -0,0 +1,103 @@
+
+**First place** for *HONEY SNOWFLAKES* goes to [**Monstrata**](https://osu.ppy.sh/users/2706438) with a total score of **190.8**! Monstrata's entry surpassed all the others by a large margin, achieving the highest scores for each category as well as the only exceptional score from judge Electoz.
+
+In **second place** is [**Kibbleru**](https://osu.ppy.sh/users/3193504) with a total score of **166.3**! Kibbleru scored consistently high in each category showcasing a overall skillful entry.
+
+And **third place** goes to [**Nozhomi**](https://osu.ppy.sh/users/2716981) with a total score of **157.5**! Nozhomi also scored consistently high in each category, notably ahead of the rest of the entries.
+
+[Click here for the complete results for *HONEY SNOWFLAKES* including judges' comments for each entry.](https://docs.google.com/spreadsheets/d/1DFpLr0JEsoWqMHdKEN8ytN4uW0mYShUpe0VTp8k-08k/edit?usp=sharing)
+  
+Thank you to everybody who submitted an entry and congratulations to all of the top three finishers! Here is what they will receive as their reward (as a reminder, 6 Olympiad Points are needed for one badge level):
+
+- **First place**: 3 months of osu!supporter, 6 Olympiad Points (resulting in 'Mapping Olympian' badge)
+- **Second place**: 2 months of osu!supporter, 4 Olympiad Points
+- **Third place**: 1 month of osu!supporter, 2 Olympiad Points
+
+As a bonus for winning first place, each of the winners' badge descriptions will include the song they placed first in!
+
+For future reference, [here is a listing of all accumulated Olympiad Points](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing) which will be updated after each contest.
+
+We'd like to thank all of the judges who spent several weeks going through and judging all of the entries: [Cerulean Veyron](https://osu.ppy.sh/users/1886524), [Electoz](https://osu.ppy.sh/users/6485263), [Kuron-kun](https://osu.ppy.sh/users/2697284), [Sinnoh](https://osu.ppy.sh/users/4236057), and [Sonnyc](https://osu.ppy.sh/users/11771). None of this would have been possible without you all.
+
+Lastly, below are all of the entries for each song for everyone to look through. In addition, pishifat, Bonzi, and Monstrata will be putting together mapsets for ranking which will include the top three entries for each of the songs they won respectively:
+
+- *Kuba Oms - Beautiful Uncertainty (Redux)*: [All entries](https://drive.google.com/open?id=1yYaGqLcZLitH9g_KxENJqOOT6PtXTFul), [Mapset](https://osu.ppy.sh/s/801774)
+- *cYsmix - Escapism*: [All entries](https://drive.google.com/open?id=10gw1hnY-JgrMMQ6a54ngX3GcwK-on6wg), [Mapset](https://osu.ppy.sh/s/801862)
+- *OISHII - HONEY SNOWFLAKES*: [All entries](https://drive.google.com/open?id=1Fh_unuEzgb1DSdOMEwgcuPVdQebn83zt), [Mapset](https://osu.ppy.sh/s/801776)
+
+The next Olympiad contest to look forward to will be for the osu!mania game mode, so be on the lookout for an announcement in the coming weeks!
```

Comment:
will be for the osu!mania game mode, -> will be for osu!mania,
adding gamemode is kinda redundant